CHANGELOG.md

```diff
@@ -4,11 +4,18 @@ x.y.z Release notes (yyyy-MM-dd)
 Xcode 12.4 is now the minimum supported version of Xcode.
 
 ### Enhancements
-* None.
+* Add support for querying on the properties of objects contained in
+  dictionaries (e.g. "dictProperty.@allValues.name CONTAINS 'a'").
+* Improve the error message for many types of invalid predicates in queries.
+* Add support for comparing `@allKeys` to another property on the same object.
 
 ### Fixed
 * Add missing `Indexable` support for UUID. 
   ([Cocoa #7545](https://github.com/realm/realm-cocoa/issues/7545), since v10.10.0)
+* `@allValues` queries on dictionaries accidentally did not require "ANY".
+* Case-insensitive and diacritic-insensitive modifiers were ignored when
+  comparing the result of an aggregate operation to another property in a
+  query.
```

Comment:
Since ?

Reply:
Whenever we first implemented collection aggregates sometime in the 0.x version range, but it's sorta unclear if it ever actually mattered before now.